src/opera/pge/cslc_s1/cslc_s1_pge.py

```diff
@@ -622,6 +651,10 @@ def _collect_cslc_product_metadata(self, cslc_product):
         # Extract all metadata assigned by the SAS at product creation time
         try:
             output_product_metadata = get_cslc_s1_product_metadata(cslc_product)
+            output_product_metadata['MeasuredParameters'] = self.augment_measured_parameters(
+                # flatten_h5_metadata(output_product_metadata)
```

Comment:
Is this comment a left-over placeholder? If so we can just remove it.

Reply:
It absolutely is a leftover from my previous approach at this